spark/src/main/scala/org/apache/spark/sql/delta/stats/DataSkippingStatsTracker.scala

```diff
@@ -87,14 +87,11 @@ class DeltaTaskStatisticsTracker(
 
   // This projection combines the intermediate results stored by aggBuffer with the values of the
   // currently processed row and updates aggBuffer in place.
-  private val updateStats: MutableProjection = GenerateMutableProjection.generate(
-    expressions = JoinedProjection.bind(
-      aggBufferAttrs,
-      dataCols,
-      aggregates.flatMap(_.updateExpressions)),
-    inputSchema = Nil,
-    useSubexprElimination = true
-  )
+  private val updateStats: MutableProjection = {
+    val expressions = JoinedProjection.bind(
+      aggBufferAttrs, dataCols, aggregates.flatMap(_.updateExpressions))
```

Comment:
Tiny nit -- while we're cleaning up the nesting, it may be worth cleaning up all the way, for maximum readability?
```suggestion
    val aggs = aggregates.flatMap(_.updateExpressions)
    val expressions = JoinedProjection.bind(aggBufferAttrs, dataCols, aggs)
```

Reply:
Done